content/docs/user-guide/basic-concepts/stage.md

```diff
@@ -2,7 +2,8 @@
 name: Stage
 match: [stage, stages]
 tooltip: >-
-  A stage represents individual data processes, including their input and
-  resulting output which can be combined to build detailed machine learning
-  pipelines.
+  A stage represents an individual data processes, including its data input(s)
```

Comment:
data process - again, not big fan of this. Can we make it less formal? E.e. represnts command or a script (list of commands)

Reply:
How about `Stages capture the commands, scripts, or code that a DVC pipeline executes, including their data input(s) and resulting output(s)`? Going with that for now.